### DISCUSSION TITLE
Allocator - No Used Allowance

### DISCUSSION THREAD

**Discussion by Kevin-FF-USA** [filecoin-project/Allocator-Governance#59] [General]
As of 10 June 2024
Allocators who have not used any of the initial 5Pib DataCap allocations

1. Web3mine | [f03019928](https://filecoinpulse.pages.dev/allocator/f03019928/)

2. Nebula Block | [f03019861](https://filecoinpulse.pages.dev/allocator/f03019861/)

3. TopBlocks | [f03019931](https://filecoinpulse.pages.dev/allocator/f03019931/#pageparamsallocator_id)

4. QC Data Center | [f03014608](https://filecoinpulse.pages.dev/allocator/f03014608/#pageparamsallocator_id)

5. FBG Capital | [f03019934](https://filecoinpulse.pages.dev/allocator/f03019934/#pageparamsallocator_id)

6. ZhongYiGuoLian | [f03019935](https://filecoinpulse.pages.dev/allocator/f03019935/#pageparamsallocator_id)

7. XnMatrix | [f03019938](https://filecoinpulse.pages.dev/allocator/f03019938/#pageparamsallocator_id)

8. SUPERB GRACE LIMITED | [f03018026](https://filecoinpulse.pages.dev/allocator/f03018026/#pageparamsallocator_id)

9. New Web Group | [f03015081](https://filecoinpulse.pages.dev/allocator/f03015081/#pageparamsallocator_id)

10. Lighthouse.Storage | [f03019939](https://filecoinpulse.pages.dev/allocator/f03019939/#pageparamsallocator_id)

11. IPFSCN | [f03019942](https://filecoinpulse.pages.dev/allocator/f03019942/#pageparamsallocator_id)

12. ANTPOOL | [f03019944](https://filecoinpulse.pages.dev/allocator/f03019944/#pageparamsallocator_id)

13. ZetaCube Inc | [f03018530](https://filecoinpulse.pages.dev/allocator/f03018530/#pageparamsallocator_id)

14.  Power Meta Corp | [f03019948](https://filecoinpulse.pages.dev/allocator/f03019948/#pageparamsallocator_id)

15. Twinquasar - CIDgravity | [f03019296](https://filecoinpulse.pages.dev/allocator/f03019296/#pageparamsallocator_id)

16. Twinquasar - CIDgravity | [f03019298](https://filecoinpulse.pages.dev/allocator/f03019298/#pageparamsallocator_id)

17. Titan Network | [f03018977](https://filecoinpulse.pages.dev/allocator/f03018977/#pageparamsallocator_id)

18. Flamenco team | [f03019950](https://filecoinpulse.pages.dev/allocator/f03019950/#pageparamsallocator_id)

**Comment by galen-mcandrew** [comment]
Discussion continued here: https://github.com/filecoin-project/Allocator-Governance/issues/6#issuecomment-2195195320

Final list of inactive Allocators who are being sunset

Nebula Block | [f03019861](https://filecoinpulse.pages.dev/allocator/f03019861/)
FBG Capital | [f03019934](https://filecoinpulse.pages.dev/allocator/f03019934/#pageparamsallocator_id)
ZhongYiGuoLian | [f03019935](https://filecoinpulse.pages.dev/allocator/f03019935/#pageparamsallocator_id)
XnMatrix | [f03019938](https://filecoinpulse.pages.dev/allocator/f03019938/#pageparamsallocator_id)
SUPERB GRACE LIMITED | [f03018026](https://filecoinpulse.pages.dev/allocator/f03018026/#pageparamsallocator_id)
Power Meta Corp | [f03019948](https://filecoinpulse.pages.dev/allocator/f03019948/#pageparamsallocator_id)
Flamenco team | [f03019950](https://filecoinpulse.pages.dev/allocator/f03019950/#pageparamsallocator_id)
